Classes/Updates/SolrSearchCTypeMigration.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the TYPO3 CMS project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+namespace ApacheSolrForTypo3\Solr\Updates;
+
+use TYPO3\CMS\Install\Attribute\UpgradeWizard;
+use TYPO3\CMS\Install\Updates\AbstractListTypeToCTypeUpdate;
+
+/**
+ * Migrates Plugin subtypes to content elements.
+ * See: https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/13.4/Deprecation-105076-PluginContentElementAndPluginSubTypes.html#deprecation-105076-plugin-content-element-and-plugin-sub-types
+ */
+#[UpgradeWizard('solrSearchCTypeMigration')]
```

Comment:
We need to enable `autoconfigure` and `autowire` for `ApacheSolrForTypo3\Solr\Updates` to ensure the wizard can be found:

```
  ApacheSolrForTypo3\Solr\Updates\:
    resource: '../Classes/Updates/*'
    autoconfigure: true
    autowire: true
```